### ISSUE TITLE
Fix CNAME file issue

### ISSUE THREAD

**Issue by deanputney** [deanputney/earlyvoting#95]
Github determines where to route requests from other domains [using a file](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/troubleshooting-custom-domains-and-github-pages#cname-errors) named `CNAME` in the repository. This is causing us problems with our current deployment system because when we run `npm run deploy` the `CNAME` file is removed (presumably during a force push). Part of the reason for moving to the subfolder structure (#94) is to avoid overwriting this file when pushing.

Basically, fixing this issue is one part of fixing our deployment system.
